guidelines/content-guidelines/09-add-documentation-topic.md

```diff
@@ -3,86 +3,85 @@ title: Add new documentation to the website
-      mode: package
-      url: https://github.com/{{ .Values.global.kymaOrgName }}/kyma/archive/{{ .Values.global.docs.clusterDocsTopicsVersion }}.zip
-      filter: /docs/api-gateway-v2/
-```
-​
+
+   ```yaml
+   apiVersion: rafter.kyma-project.io/v1beta1
+   kind: ClusterAssetGroup
+   metadata:
+     labels:
+       rafter.kyma-project.io/view-context: docs-ui
+       rafter.kyma-project.io/group-name: components
+       rafter.kyma-project.io/order: "11"
+     name: api-gateway-v2
+   spec:
+     displayName: "API Gateway v2"
+     description: "Overall documentation for API Gateway v2"
+     sources:
+       - type: markdown
+         name: docs
+         mode: package
+         url: https://github.com/{{ .Values.global.kymaOrgName }}/kyma/archive/{{ .Values.global.docs.clusterAssetGroupVersion }}.zip
+         filter: /docs/api-gateway-v2/
+   ```
+
 3. Adjust values for these fields:
-​
-- **cms.kyma-project.io/order** defines the number of your topic on the list in the left navigation, such as `"11"`. To add it correctly, check in other `.yaml` files which number is assigned to the last documentation topic in the navigation on the website, and add a consecutive number to your component. If you decide to modify the existing topic order, change values for this parameter in all other `.yaml` files accordingly to avoid duplicates.
+
+- **rafter.kyma-project.io/order** defines the number of your topic on the list in the left navigation, such as `"11"`. To add it correctly, check in other `.yaml` files which number is assigned to the last documentation topic in the navigation on the website, and add a consecutive number to your component. If you decide to modify the existing topic order, change values for this parameter in all other `.yaml` files accordingly to avoid duplicates.
 - **metadata.name** defines the CR name, such as `api-gateway-v2`.
 - **spec.displayname** defines the component name displayed on the website, such as `"API Gateway v2"`.
 - **spec.sources.filter** defines the location of the new topic's document sources, such as `/docs/api-gateway-v2/`.
-​
+
 4. Merge the changes and wait until the website is rebuilt.
 
 ## Add a single OpenAPI specification
-​
+
 In addition to Kyma documentation, there are also [OpenAPI](https://swagger.io/specification/) specifications rendered on the [Kyma website](https://kyma-project.io). You can find these specifications under the **API Consoles** type in the right navigation panel of a given component topic.
 ​
-
 To add a new specification, follow these steps:
-​
 
-1. Go to the [`templates`](https://github.com/kyma-project/kyma/tree/master/resources/core/charts/docs/charts/content-ui/templates) folder and locate an existing ClusterDocsTopic CR that you want to modify.
+1. Go to the [`templates`](https://github.com/kyma-project/kyma/tree/master/resources/core/charts/docs/charts/content-ui/templates) folder and locate an existing ClusterAssetGroup CR that you want to modify.
```

Comment:
Path will change:

```md
1. Go to the [`templates`](https://github.com/kyma-project/kyma/tree/master/resources/core/charts/content-ui/templates) folder and locate an existing ClusterAssetGroup CR that you want to modify.
```

Reply:
Link `https://github.com/kyma-project/kyma/tree/master/resources/core/charts/docs/charts/content-ui/templates` is correct. Sorry.